documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md

```diff
@@ -0,0 +1,136 @@
+#  Agenda and Minutes of CAMARA Technical Steering Committee (TSC) meeting - 2023-10-19
+
+## Invited TSC Participants
+
+* [x] Herbert Damker - (Deutsche Telekom AG) Active Maintainer, TSC chair
+* [x] Eric Murray - (Vodafone) Active Maintainer, TSC deputy chair
+* [x] Ludovic Robert - (Orange) Active Maintainer, TSC deputy chair
+* [x] Adnan Saleem - (Radisys) End User Council representative
+* [ ] Chris Howell - (Vonage) Active Maintainer
+* [x] Diego González Martínez - (Telefónica) Active Maintainer
+* [x] Doug Makishima - (Summit Tech) End User Council representative
+* [ ] George Glass - TM Forum representative
+* [ ] Henry Calvert - GSMA representative
+* [x] Mark Cornall - GSMA representative
+* [x] Jan Friman - (Ericsson) Active Maintainer
+* [x] Jose Luis Urien - (Telefonica) Active Maintainer
+* [ ] Kevin Smith - (Vodafone) Active Maintainer
+* [x] Mahesh Chapalamadugu - (Verizon) End User Council representative
+* [x] Shilpa Padgaonkar - (Deutsche Telekom AG) Active Maintainer
+* [x] Toshi (Toshiyasu) Wakayama - (KDDI) Active Maintainer
+
+Guest Presenter:
+
+* Dawid McBride (Linux Fundation)
+
+Attendees:
+*  Sylvain Morel (Orange)
+*  Helene Vigue (GSMA)
+*  Ivan Izaquirre MArtin (Telefonica)
+*  Dinesh Muthukumar
+*  Mehdi Jafarizadeh
+*  Pierre Close (AT&T)
+*  Casey Cain (Linux Fundation)
+*  Tanja de Groot (Nokia)
+
+
+Minutes are available within: [https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC](https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC)
+
+## Agenda
+
+* Introduction
+* Release Management (#82) 
+* Commonalities & ICM 
+    * Alignment around securitySchemes and security of CAMARA API specs
+* API Backlog 
+    * New proposal for review "Network Insights"
+    * Decision on API Family for "Device Swap"
+    * Status of Site-to-cloud VPN proposal
+* (Further) open issues in https://github.com/camaraproject/Governance/issues
+* AOB
+
+Please find material for the agenda points below.
+
+## Introduction
+
+The participation of TSC Members covers well all CAMARA subproject.
+
+## Release Management
+
+A deck was prepared by Ludovic/Herbert (Available within meeting notes folder)
+
+Following points were discussed:
+
+**Discussion on API versioning** – DT proposal to use alpha<n> in API path until and API (major) version is stable
+  * Distinction between repostiory release (semantic version as before) and the version shown within API base path
+    * A sub project repository can contain one or multiple (closely related) APIs
+    * The version number of releases on sub project level will be used within all APIs within the repository (= the API release belongs to vX.Y.Z of the sub project)

```

Comment:
This sentence is not clear for me. I recall we said that not every API will have the same version of the release, but release will match the most advanced API in sub-project. E.g.: Release 3.2.1 with API-X in version 3.2.1, API-Y in version v2alpha1 plus in version v1.0.2, etc

Reply:
Agreed - reformulated.

Reply:
I suppose we need a more elaborated concept to discuss the details. The sentence was originally meant that there will be full semantic versioning only on the release of the sub project. 

But indeed, such release can contain APIs which have different major versions within the base path. To give an example:

Release 2.x.y of Device Location can contain (more complex then it most probably will be in practice, just to show the principles):
- location-verification/v1 (the first stable version, if still supported, not anymore evolved)
- location-verification/v2 (the second stable version, due to breaking changes a different API)
- location-verification/v2alpha3 (alpha version with enhancements compared to v2 which potentially will be added to the stable version within a next release of DeviceLocation)
- location-retrieval/v1 (stable version)
- location-retrieval/v2alpha1 (first alpha of next major version of location-retrieval, new major number due to lack of backward-compatibility to location-retrieval/v1)

All these API versions are belonging to the same release of the repository DeviceLocation and are unambigious defined by the commit the release is pointing to.

Reply:
@hdamker : I did not added this comment in the minutes as for me this comment should be part of the release management guidance document that you will provided - is this correct?

Reply:
API versions need to have also vx.y.x even if URL path has only major version. Anyway, I think agreed way forward in TSC is clear in minutes and we should finetune the details in the document.